### ISSUE TITLE
PC recorded video not play in iphone browser

### ISSUE THREAD

**Issue by puspendra09** [0x006F/react-media-recorder#81]
when I record video in chrome browser using pc and stored in s3 bucket, that video play with pc browser but in iphone browser occur an error. Have an any solution to resolve this issue to play video with independ platforms.

**Comment by github-alethea** [comment]
If you could share more details regarding your issue then would be helpful.
But anyway, this is a common issue on iPhone devices, to make it work, the recorded video has to be encoded to H.264 for the iPhone devices and use the default one for other platforms.
You could code your own solution on your own backend service to encode it or you could use AWS [mediaconvert](https://aws.amazon.com/mediaconvert/) service to do so as you are already storing it on S3.

**Comment by github-id01** [comment]
If you could share more details regarding your issue then that would be helpful.
But anyway, this is a common issue on iPhone devices, to make it work, the recorded video has to be encoded to H.264 for the iPhone devices and use the default one for other platforms.
You could code your own solution on your own backend service to encode it or you could use AWS mediaconvert service to do so as you are already storing it on S3.

**Comment by eli-s-r** [comment]
@github-id01 Do you know of any way to do this client-side without using a large package such as https://github.com/Kagami/ffmpeg.js? We need to support immediate playback of (audio) recordings in the browser, _before_ the contents are sent to the server.

**Comment by github-id01** [comment]
Nope, I won't recommend such a thing to do on the client-side, a back-end service would be better suited for such a task,
